lib/vmfloaty/utils.rb

```diff
@@ -206,6 +206,9 @@ def self.get_service_config(config, options)
         # If the service is configured but some values are missing, use the top-level defaults to fill them in
         service_config.merge! config['services'][options.service]
       end
+    # No config file but service is declared on command line
+    elsif config['services'].nil? && options.service
```

Comment:
Small nitpick on this, but it's simpler to check for nil like this!

```suggestion
    elsif !config['services'] && options.service
```